### PR TITLE
Added a simple package and added metadata

### PR DESCRIPTION
**Pull request by ykarbaschi**
Please only look at minimalPackage.py

### CODE REVIEW

File: MakeQuiltPackage/minimalPackage.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import quilt
+import pandas as pd
+import os
+import shutil
+import pathlib
+
+############################################
```
Comment (akarve) [inline MakeQuiltPackage/minimalPackage.py]: this is nonstandard

File: MakeQuiltPackage/minimalPackage.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import quilt
+import pandas as pd
+import os
+import shutil
+import pathlib
```
Comment (akarve) [inline MakeQuiltPackage/minimalPackage.py]: usually imports are alphabetized by import and broken into 3 sections: stdlib, third party, local deps

File: MakeQuiltPackage/minimalPackage.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+############################################
+def GenerateREADME(readmePath):
+	with open(readmePath,'w') as myFile:
+		myFile.write('# Package Summary')
+
+def GeneratePandaTable(imageIDList, annotationDF, metadataPath):
```
Comment (akarve) [inline MakeQuiltPackage/minimalPackage.py]: nitpick: "Pandas" not "panda"

File: MakeQuiltPackage/minimalPackage.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+#########################
+
+dataFolder = './minPackage'
+imageIDList = ['b91d35b5b1e21b3a', 'b91e1150fafff17c', 'b91f356627cc68b1']
+
+GenerateREADME(dataFolder+'/README.md')
+from quilt.data.examples import openimages as oi
+annotationDF = oi.train.annotations_human()
+GeneratePandaTable(imageIDList, annotationDF, dataFolder + '/PandaTable.csv')
```
Comment (akarve) [inline MakeQuiltPackage/minimalPackage.py]: not sure why you're doing this? if it's to put the table into the package you can just call `_set` directly on a dataframe

File: MakeQuiltPackage/minimalPackage.py [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+
+if (os.path.exists(dataFolder+'/build.yml')):
+	os.remove(dataFolder+'/build.yml')
+
+quilt.generate(dataFolder)
+quilt.build("ykarbaschi/minimalPackage", dataFolder+'/build.yml')
+
+from quilt.data.ykarbaschi import minimalPackage
+minimalPackage._meta['trainable'] = True
```
Comment (akarve) [inline MakeQuiltPackage/minimalPackage.py]: this is hardwired. why?

note: you'll need an as yet unreleased branch from Dima for this to work, but it looks good.

File: MakeQuiltPackage/minimalPackage.py [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+minimalPackage._meta['bbox'] = True
+minimalPackage._meta['labelName'] = 'Minimal Package'
+minimalPackage._meta['image_count'] = 3
```
Comment (akarve) [inline MakeQuiltPackage/minimalPackage.py]: i guess OK for building one package but why hardwired? do more images so you can take some reasonable measurements. make this package real but for one label.

File: MakeQuiltPackage/minimalPackage.py [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+for imageID in imageIDList:
+	imageDataNode = getattr(getattr(minimalPackage, 'images'), imageID)
+	try:
+		imageDataNode._meta['bbox'] = pd.DataFrame(data=['for example bbox information'])
```
Comment (akarve) [inline MakeQuiltPackage/minimalPackage.py]: indentation looks weird; 4 is standard for Python; see https://google.github.io/styleguide/pyguide.html

File: MakeQuiltPackage/minimalPackage.py [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+	except Exception as e:
+		pass
```
Comment (akarve) [inline MakeQuiltPackage/minimalPackage.py]: it's a bad idea to squelch exceptions. then something bad happens and you don't know about it. at least print to stderr.

File: MakeQuiltPackage/minimalPackage.py [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+
+quilt.login()
```
Comment (akarve) [inline MakeQuiltPackage/minimalPackage.py]: you probably don't want this in the script. just `$ quilt login` first and then that machine is logged in

File: MakeQuiltPackage/minimalPackage.py (hunk 5 above)
Comment (ykarbaschi) [reply MakeQuiltPackage/minimalPackage.py]: what do you mean by hardwired?

File: MakeQuiltPackage/minimalPackage.py (hunk 4 above)
Comment (ykarbaschi) [reply MakeQuiltPackage/minimalPackage.py]: for example: `pkgName._set(['_meta'], PandaTable)`?

**Pull request merged**